features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -70,6 +70,9 @@
         <TrustManagerType>{{key_mgt.trust_manager_type}}</TrustManagerType>
     </Security>
 
+    <!--This configuration is to resolve the internal service URL for internal API calls-->
```

Comment:
add space after `<!--` and before` -->`

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2949/commits/0c97e69333943613c0cac9c3e6dd5c2f1eab66fd